### PR TITLE
Make users' workshop dates less ambiguous.

### PR DESCRIPTION
**Pull request by mtbc**
Easier for an international audience to read at a glance. Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/omero-users-workshop-dundee-april-2019.html and https://snoopycrimecop.github.io/www.openmicroscopy.org/events/omero-users-workshop-dundee-june-2019.html.

### CODE REVIEW

File: events/omero-users-workshop-dundee-april-2019.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h2 class="event-day">Location: Dundee</h2>
         </div>
         <div class="row">
             <div class="column">
-                <h2 class="event-day">Date: 08-09/04/2019</h2>
+                <h2 class="event-day">Date: 8th-9th April 2019</h2>
```
Comment (sbesson) [inline events/omero-users-workshop-dundee-april-2019.html]: https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ gives `April 08-09, 2019`. Worth unifying the date format ?

Comment (mtbc) [reply events/omero-users-workshop-dundee-april-2019.html]: I don't think so: the other events aren't unified and the context is different. (The index is a tiny note next to the calendar icon, this page is rather more verbose about the event.)

**Comment by mtbc** [comment]
> or switching to ISO-8601

I do like to use ISO-8601 but in this case if I were going for a less-minimal change then I'd probably use a format that includes day of week: that's useful extra information for figuring meetings/conferences. That's probably a less urgent concern though.

**Comment by mtbc** [comment]
I have been thinking about how most nicely to include day of week in date ranges. What would you think of something like,

> Date: Monday 8th to Tuesday 9th April 2019

Too verbose for this page?

No rush of course.

**Comment by sbesson** [comment]
Sorry, I wasn't sure you were investigating date formats. I primarily merged this as my comment was not particularly a blocker and I wonder whether the training team wanted to redeploy the website before Monday's event /cc @jburel @pwalczysko 

Looking at date formats used by external training events, I like http://www.nibb.ac.jp/abis/gbi-abis_training?lang=en which is quite similar to your proposal.

**Comment by mtbc** [comment]
Sure, no problem, made sense to merge this PR, can always revise at our leisure.

Maybe flip around to US ordering too then; I wasn't sure about that as most of our training is European anyway.

**Comment by jburel** [comment]
We have tried to keep it not too long on that page 
For Date format, so far very limited session in US but I guess the format needs to match the rest of the website

**Pull request merged**